src/service/iot_config.proto

```diff
@@ -346,14 +353,25 @@ message gateway_load_region_req_v1 {
   bytes signature = 4;
 }
 
-message gateway_load_region_res_v1 {}
+message admin_load_region_res_v1 {}
 
-message gateway_location_req_v1 {
-  bytes gateway = 1;
+message admin_add_key_req_v1 {
+  enum key_type_v1 {
+    operator = 0;
+    packet_router = 1;
+  }
+
+  string pubkey = 1;
```

Comment:
So pub keys are still passed around as bytes and not as strings right?

Reply:
yeah, i was thinking of this only from the perspective of the operator using this but bytes is consistent across all the other uses of passing around a pubkey and it's easy enough to base64 encode them

Reply:
fixed [ca3da06](https://github.com/helium/proto/pull/288/commits/ca3da06d51faa7b490e2377cf47c20098e9900c5)

Reply:
the user passes it to the cli as a public key (b58 string). The cli would then decode it.. you're not base64 encoding keys now are you?!

Reply:
You’re right, b58, not 64